src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -60,6 +66,12 @@ export class MarqueeWizardComponent<TStepContext = any> extends WizardComponent
             .subscribe(this.onResize.bind(this));
     }
 
+    ngAfterViewInit(): void {
+        setTimeout(() => {
+            this.tabbleList.setFirstItemTabbable();
+        }, 50);
```

Comment:
It looks suspiciously like you've fudged this number, what exactly is happening during this time period and how do you know that it will never take more than 50ms?